copilot/data/download.go

```diff
@@ -25,7 +25,7 @@ import (
 
 type Downloader struct {
 	format core.DataLoadingConfig_LiteralMapFormat
-	store *storage.DataStore
+	store  *storage.DataStore
```

Comment:
Format?

Reply:
yep, i ran `make goimports` but I can revert though if it's annoying to mess with git blame